superset/assets/src/dashboard/util/publishSubscriberUtil.js

```diff
@@ -30,17 +32,62 @@ export function getUniqueActionsForSlice(slice) {
 export function getModalSliceIDFor(publishSubscriberMap, publisherId) {
     if (publishSubscriberMap && publishSubscriberMap.hasOwnProperty('subscribers')) {
         const subscribers = Object.values(publishSubscriberMap.subscribers);
-        var item = _.find(subscribers, function (item) {
+        var item = find(subscribers, function (item) {
             return ((item.actions.indexOf(USE_AS_MODAL) > -1) && (Object.keys(item.linked_slices).indexOf(publisherId) != -1));
         });
         return item ? item.id : undefined;
     }
     return undefined
 }
 
+export function getSlicesWithSubHeader(subscribers, chartId, filters) {
+    let subHeader = '';
+    if (chartId != -1 && subscribers && subscribers[chartId]) {
+        const subscriber = subscribers[chartId];
+        if (subscriber.actions.indexOf(INCLUDE_IN_TITLE) > -1) {
+            subHeader = '';
+            for (let lsKey in subscriber.linked_slices) {
+                if (keyExists(lsKey, filters)) {
+                    let linkedSlicesLength = subscriber.linked_slices[lsKey].length;
+
+                    for (let i = 0; i < linkedSlicesLength; i++) {
+                        let linkedSlice = subscriber.linked_slices[lsKey];
+
+                        if (linkedSlice[i].actions.indexOf(INCLUDE_IN_TITLE) > -1) {
+                            let columnName = linkedSlice[i]['col'];
+                            let subHeaderValues = filters[lsKey][columnName];
+                            const values = subHeaderValues ? subHeaderValues.constructor == Array ? subHeaderValues.concat() : [subHeaderValues] : [];
+
+                            if (subHeaderValues) {
```

Comment:
condition should be values length not subHeaderValues because subHeaderValues can be blank....

Reply:
`values` is already taken care in next line and `subHeaderValues` cannot be blank.